### PR TITLE
Adds *inhale and *exhale

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by coiax** [comment]
Nice commit history. :fire:

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -166,4 +166,13 @@
 		dna.species.mutant_bodyparts |= "wings"
 	update_body()
 
+/datum/emote/living/carbon/human/inhale
+	key = "inhale"
+	message = "inhales"
```
Comment (coiax) [inline code/modules/mob/living/carbon/human/emote.dm]: Shouldn't it be "inhales."

Comment (vcordie) [inline code/modules/mob/living/carbon/human/emote.dm]: its a meme, don't take it too seriously

Comment (coiax) [inline code/modules/mob/living/carbon/human/emote.dm]: I take my memes very seriously, have you seen the Hyperfractal Gigashuttle?

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -166,4 +166,13 @@
...
 	update_body()
 
+/datum/emote/living/carbon/human/inhale
+	key = "inhale"
+	message = "inhales"
+
+/datum/emote/living/carbon/human/exhale
+	key = "exhale"
+	message = "exhales"
+
```
Comment (Supermichael777) [inline code/modules/mob/living/carbon/human/emote.dm]: No losebreath?

**Comment by jughu** [comment]
please give a reason why you want this added to the game - kor

**Comment by MrPerson** [comment]
meme meme meme
meme
meme meme meme, meme meme

**Comment by PKPenguin321** [comment]
say *inhale
say boi

**Pull request closed without merging**